opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/controller/PsuAuthController.java

```diff
@@ -102,6 +108,23 @@
         return Optional.empty();
     }
 
+    @Override
+    public ResponseEntity<Void> renewalAuthorizationSessionKey(UUID xRequestId, UUID authorizationId) {
+        if (!authorizationKeyFromHttpRequest.isPresent()) {
+            log.warn("No Cookie provied, call not accepted");
```

Comment:
Better to
`log.warn("Authorization session key renewal call failed for {} authorization - missing cookie", authorizationId);`

Reply:
done